apis/clusters/v1beta1/cassandra_webhook.go

```diff
@@ -176,3 +167,69 @@ func (cv *cassandraValidator) ValidateDelete(ctx context.Context, obj runtime.Ob
 	// TODO(user): fill in your validation logic upon object deletion.
 	return nil
 }
+
+func (cv *cassandraValidator) validateDataCentresUpdate(new, old *Cassandra) error {
+	for _, newDC := range new.Spec.DataCentres {
```

Comment:
`ErrDecreasedDataCentresNumber` is missing

Reply:
Fixed.